### DISCUSSION TITLE
problems with workflow.py for 1.0.23

### DISCUSSION THREAD

**Discussion by ppimenta0** [ronshnapp/MyPTV#48] [Q&A]
Hello everyone,

I am trying to use the new calibration model, but while using the workflow script, I keep getting import errors (as in they are not in the folder specified). Problems started with final_calibration. While segmentation worked well, matching had the same problem - camera is not being imported. Other functions, such as get_epipolar are giving errors. I looked at the dependencies and it seemed like maybe some legacy functions were being called? 
If anybody experienced the same problems, I would appreciate any help.

Best,

Pedro
SIO/UCSD

**Comment by ronshnapp** [comment]
Hi Pedro,

That's strange. From what i understand, some of the modules are imported but others are not. Are you using the workflow file that came with the new version?

Also, could you share a screenshot with the errors you're getting? 

Thanks.

Ron
